speed-dreams/source-2.2.3/src/libs/tgfclient/tgfclient.h

```diff
@@ -456,6 +456,9 @@ TGFCLIENT_API int GfuiVisibilitySet(void* scr, int id, int visible);
 TGFCLIENT_API int GfuiEnable(void* scr, int id, int flag);
 TGFCLIENT_API void GfuiUnSelectCurrent(void);
 
+// SIMULATED DRIVING ASSISTANCE: expose the GfuiFontClass to the API
+class TGFCLIENT_API GfuiFontClass;
```

Comment:
add to CHANGES.txt

Reply:
Already in there (see last commit before the PR opened for review)